etc/profile-a-l/file-roller.profile

```diff
@@ -6,6 +6,9 @@ include file-roller.local
 # Persistent global definitions
 include globals.local
 
+# Allow dpkg (blacklisted by disable-common.inc)
```

Comment:
```suggestion
```

For consistency; such comments are usually only used when including allow-.inc
files.